cypress/support/pageBuilder/pbCreateCategoryAndBlocks.js

```diff
@@ -0,0 +1,102 @@
+import { GraphQLClient } from "graphql-request";
+
+function makeid(length) {
+    var result = "";
+    var characters = "abcdefghijklmnopqrstuvwxyz";
+    var charactersLength = characters.length;
+
+    for (var i = 0; i < length; i++) {
+        result += characters.charAt(Math.floor(Math.random() * charactersLength));
+    }
+    return result;
+}
+
+Cypress.Commands.add("pbCreateCategoryAndBlocks", (categoryVariables, numBlocks) => {
+    cy.login().then(user => {
+        const client = new GraphQLClient(Cypress.env("GRAPHQL_API_URL"), {
+            headers: {
+                authorization: `Bearer ${user.idToken.jwtToken}`
+            }
+        });
+
+        const createCategoryMutation = `
```

Comment:
It might've been better to have the `pbCreateCategoryAndBlocks` be composed from `pbCreateBlockCategory` and `pbCreateBlock`, because then we'd be able to use these cmds separately in new tests. But I guess we can leave this as is for now, and refactor when/if needed.

Reply:
Well we do have `pbCreateBlock.js` and `pbCreateCategory.js`.

I just combined the two into one to create `pbCreateCategoryAndBlocks.ts`.